src/test/ui/macros/macro-in-expression-context-2.stderr

```diff
@@ -1,4 +1,4 @@
-error: expected expression, found `<eof>`
+error: expected expression, found the end of the macro arm
```

Comment:
"the macro arm" seems like an odd turn of phrase to me. I'm not entirely sure what it is referring to, to be honest. Something in the macro definition?

Reply:
Well, in this case it should be pointing at

```
macro_rules! empty { () => () }
                            ^ the end of the macro arm
```
while I'm calling
```
macro_rules! empty { () => () }
                           ^^ the macro arm
```

Is there an appropriate name for it? Because to me the end of the macro definition would be
```
macro_rules! empty { () => () }
                              ^ the end of the macro def
```

Reply:
That seems like an ok term. I expect "macro arm" to refer to something in the macro definition, which it seems like it does. I'm not sure if users will understand it per se, since I don't know how common it is to talk about "arms" etc... but maybe they can intuit =) 

I don't really understand that error tbh -- there is nothing wrong with that macro definition, is there? Oh, I guess ... if it is used?

Wow,

```rust
macro_rules! foo {
    () => ()
}

fn main() {
    foo!()
}
```

gives a truly horrible error message...

Reply:
I feel like I would like to see an error like "expected macro to expand to an expression, but macro expanded to nothing"